lib/GLPI/Agent/Task/Inventory/Generic/Remote_Mgmt/TeamViewer.pm

```diff
@@ -16,15 +16,25 @@ sub isEnabled {
 
         GLPI::Agent::Tools::Win32->use();
 
-        my $key = getRegistryKey(
-            path => is64bit() ?
-                "HKEY_LOCAL_MACHINE/SOFTWARE/Wow6432Node/TeamViewer" :
-                "HKEY_LOCAL_MACHINE/SOFTWARE/TeamViewer",
-            # Important for remote inventory optimization
-            required    => [ qw/ClientID/ ],
-            maxdepth    => 1,
-            logger => $params{logger}
+         #Depending on the installation the teamviewer key can be in two place in X64 OS
+        
+        my $key = getRegistryKey(                
+                path => "HKEY_LOCAL_MACHINE/SOFTWARE/TeamViewer",
+                # Important for remote inventory optimization
+                required    => [ qw/ClientID/ ],
+                maxdepth    => 1,
+                logger => $params{logger}                
         );
+        if(!$key){
```

Comment:
You should check OS is 64 bits before looking for `Wow6432Node` node an remove spaces at the end of the lines.

```suggestion
        my $key = getRegistryKey(
            path => "HKEY_LOCAL_MACHINE/SOFTWARE/TeamViewer",
            # Important for remote inventory optimization
            required    => [ qw/ClientID/ ],
            maxdepth    => 1,
            logger      => $params{logger}
        );
        if (!$key && is64bit()) {
```